site/shared/connectors/metaMask.md

```diff
@@ -36,133 +36,46 @@ export const config = createConfig({
-
-
-### forceInjectProvider <Badge type="warning">deprecated</Badge>
-
-`boolean | undefined`
-
-If true, the SDK will force inject the provider into the global `window` object.
-
-### infuraAPIKey
-
-`string | undefined`
+Metadata about the dapp using the SDK, including `name`, `url`, and `iconUrl`.
+By default, `name` is set to `'wagmi'`.
 
-The Infura API key to use for RPC requests.
-
-### injectProvider <Badge type="warning">deprecated</Badge>
-
-`boolean | undefined`
+```ts-vue
+import { metaMask } from '{{connectorsPackageName}}'
 
-If true, the SDK will inject the provider into the global `window` object.
+const connector = metaMask({
+  dappMetadata: { // [!code focus]
+    name: 'My Wagmi App', // [!code focus]
+    url: 'https://example.com', // [!code focus]
+    iconUrl: 'https://example.com/favicon.ico', // [!code focus]
+  }
+})
+```
 
 ### logging
 
 `SDKLoggingOptions | undefined`
 
 Options for customizing the logging behavior of the SDK.
 
-### modals
-
-`RemoteConnectionProps['modals'] | undefined`
-
-An object that allows you to customize or translate each of the displayed modals. See the nodejs example for more information.
-
-### openDeeplink
-
-`((arg: string) => void) | undefined`
-
-A function that will be called to open a deeplink to the MetaMask Mobile app.
-
-### preferDesktop
-
-`boolean | undefined`
-
-If true, the SDK will prefer the desktop version of MetaMask over the mobile version.
-
-### shouldShimWeb3
+### headless
 
 `boolean | undefined`
 
-If true, the SDK will shim the window.web3 object with the provider returned by the SDK (useful for compatibility with older browser).
-
-### storage
-
-`StorageManagerProps | undefined`
-
-Options for customizing the storage manager used by the SDK.
-
-### timer
-
-`any | undefined`
-
-A timer object to use for scheduling tasks.
-
-### transports
+Enables or disables headless mode.
+Setting this to `true` allows you to [display custom modals](https://docs.metamask.io/wallet/how-to/display/custom-modals/).
+The default is `false`.
 
-`string[] | undefined`
-
-An array of transport protocols to use for communication with the MetaMask wallet.
-
-### ui
-
-`SDKUIOptions | undefined`
-
-Options for customizing the SDK UI.
-
-### useDeeplink <Badge type="warning">deprecated</Badge>
-
-`boolean | undefined`
-
-- If `true`, the SDK will use deeplinks to connect with MetaMask Mobile. If `false`, the SDK will use universal links to connect with MetaMask Mobile.
-- Defaults to `true`.
-
-::: warning
-Setting `useDeeplink` to `false` can negatively impact performance on iOS Safari as Universal Link connections are canceled if they do not occur within ~500ms of an interaction (e.g. button press).
-:::
-
-### wakeLockType
-
-`WakeLockStatus | undefined`
+```ts-vue
+import { metaMask } from '{{connectorsPackageName}}'
 
-The type of wake lock to use when the SDK is running in the background.
+const connector = metaMask({
+  headless: true // [!code focus]
+})
+```
```

Comment:
## Advanced
By default, this connector will replace the EIP_6963 MetaMask injected connector if it is detected (See the option [`rdns`](https://wagmi.sh/dev/creating-connectors#properties))